src/main/java/net/fabricmc/loom/api/mappings/layered/spec/FileSpec.java

```diff
@@ -63,6 +64,10 @@ static FileSpec create(Object o) {
 		Objects.requireNonNull(o, "Object cannot be null");
 
 		if (o instanceof CharSequence s) {
+			if (s.toString().startsWith("https://")) {
```

Comment:
Maybe support for `http://` too (though no one should be using that), as well as `URL`s?

Reply:
Only reason I didn’t was because gradle doesn’t allow it by default for maven repos. No major disadvantage especially as this is only used for mappings?

Reply:
Ah I see. `URL` object support for this protocol would still be nice, although I don't expect that many people to use it